manifest.json

```diff
@@ -121,6 +122,7 @@
   ],
   "background": {
     "scripts": [
+      "js/language.js",
```

Comment:
I had to put this as background script, in order to access `Localization` for the context menu translations.